Gemfile

```diff
@@ -24,6 +24,7 @@ gem 'jbuilder', '~> 2.7'
 gem 'devise'
 # Use Active Storage variant
 # gem 'image_processing', '~> 1.2'
+gem 'dotenv-rails', groups: [:development, :test]
```

Comment:
What is this one?

Reply:
This is the gem to protect the cloudinary key